packages/simcore-sdk/src/simcore_sdk/node_ports_v2/nodeports_v2.py

```diff
@@ -156,26 +173,42 @@ async def set_multiple(
 
         raises ValidationError
         """
+
+        async def _set_with_notifications(
+            port_key: ServicePortKey,
+            value: ItemConcreteValue | None,
+            set_kwargs: SetKWargs | None,
+            sub_progress: ProgressBarData,
+        ) -> None:
+            assert outputs_callbacks is not None  # nosec
+            try:
+                # pylint: disable=protected-access
+                await self.internal_outputs[port_key]._set(  # noqa: SLF001
+                    value, set_kwargs=set_kwargs, progress_bar=sub_progress
+                )
+                await outputs_callbacks.finished_succesfully(port_key)
+            except UnboundPortError:
+                # not available try inputs
+                # if this fails it will raise another exception
+                # pylint: disable=protected-access
+                await self.internal_inputs[port_key]._set(  # noqa: SLF001
+                    value, set_kwargs=set_kwargs, progress_bar=sub_progress
+                )
+            except CancelledError:
+                await outputs_callbacks.aborted(port_key)
+                raise
+            except Exception:
+                await outputs_callbacks.finished_with_error(port_key)
```

Comment:
Q: So if there is an **unexpected ** error, you want to notify simply that it failed?  Since this reaches the user , I would use the technique of logging the error on the server side and sending an OEC to the front-end.

wait, you are reraising it ... so perhaps report it when you catch it?
BTW why do not you need to reraise?